src/4.3_to_5.0/migrateKpi.ts

```diff
@@ -206,6 +213,29 @@ export function migrateKpi(
     }),
   };
 
+  try {
+    const migratedTitles = getMigratedKpiTitles(legacyKpiState, {
+      cube,
+      mapping,
+    });
+    if (migratedTitles && Object.keys(migratedTitles).length > 0) {
+      migratedWidgetState.titles = migratedTitles;
+    }
+  } catch (error) {
+    // Migrating the KPI titles is a best effort.
+    // The migration script should not fail if this part errors.
+    console.warn(
+      `Could not migrate the titles of the featured values widget named "${legacyKpiState.name}". Underlying error:\n`,
+      error,
+    );
```

Comment:
It would be nice to add this to the error report instead of console.logging IMO

Reply:
I agree. I initially did not see how to do it without refactoring, but it seems that I can do it using a `PartialMigrationError`.